### ISSUE TITLE
Custom report text does not get saved/sent

### ISSUE THREAD

**Issue by laureenas** [gtimelog/gtimelog#79]
Customization added to report text do not get sent. Only the generated text gets sent instead.

Repro on commit 552493388357d57c30ae3350313a4ffb36b1973d on branch `master`

Select 'Report' from the menu.
Select 'Week' from the calendar type.
Navigate back several weeks.
Add custom text at the beginning of the report in the report window.
Click 'Send'.

A generated report text get sent, all customizations to the text are dicarded.

**Comment by mgedmin** [comment]
I cannot reproduce.

**Comment by mgedmin** [comment]
Yup, pdb on Ubuntu 14.04 confirms that `self.report_text.body` is different from `self.report_get.get_buffer().get_property('text')`.

**Comment by mgedmin** [comment]
This would be easier to fix if I had a less painful way to test gtimelog on Ubuntu 14.04.  Currently I use [testdrive](https://wiki.ubuntu.com/QATeam/Testdrive) to fetch an ISO and launch a live session in a VM.  Then I have to apt-get install git and everything else (python-gobject, gettext, ...), then git clone gtimelog, then run it.

I've tried Docker (which [used to almost-work for testing Memgraphinator](https://github.com/mgedmin/memgraphinator/blob/master/testing/trusty/Dockerfile)), but it failed horribly during `docker build` -- couldn't resolve archive.ubuntu.com while running `apt-get update`.

I've tried Vagrant (using X11 forwarding) -- GUI apps seem very broken, e.g. window is not redrawn after resizes.  This is not specific to gtimelog; other apps like Gedit or Nautilus [are affected too](http://imgur.com/1e4xJAU).
